### DISCUSSION TITLE
Permission to upload script

### DISCUSSION THREAD

**Discussion by TheDragonShaman** [RandomNinjaAtk/arr-scripts#305] [General]
Hi @RandomNinjaAtk, I really enjoyed your arr-scripts particularly for Lidarr. But they aren't available outside of containers. I have always wanted to try my hand at coding so I rewrote the AutoArtistAdder feature in python for use on Windows. I have got it working and I do little updates to it ever so ofter to clean stuff up as I learn. But so far I've only used it personally 
and havent shared.  Would you have an issue if I uploaded this to my github? I obviously plan to credit you since it was your design first.

**Comment by RandomNinjaAtk** [comment]
No issue with that!

**Reply by TheDragonShaman** [reply]
Thankyou!
